### ISSUE TITLE
Several test fail with pandas-0.17 -- ImportError: No module named 'pandas.version'

### ISSUE THREAD

**Issue by jlec** [statsmodels/statsmodels#2666]
```python
======================================================================
ERROR: statsmodels.tsa.base.tests.test_base.test_predict_freq
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/base/tests/test_base.py", line 34, in test_predict_freq
    assert_equal(predict_dates, expected_dates)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.formula.tests.test_formula.test_formula_labels
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/formula/tests/test_formula.py", line 92, in test_formula_labels
    assert_equal(model.fittedvalues.index, dta.index)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_reset_trend
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 470, in test_reset_trend
    assert_equal(len(res1.params), len(res2.params)+1)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arima_predict_mle_dates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 685, in test_arima_predict_mle_dates
    assert_equal(res1.data.predict_dates, cpi_dates[start:end+1])
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arma_predict_mle_dates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 714, in test_arma_predict_mle_dates
    assert_equal(mod.data.predict_dates, sun_dates[start:end+1])
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arima_predict_css_dates
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 744, in test_arima_predict_css_dates
    assert_equal(res1.data.predict_dates, cpi_dates[start:end+1])
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arma_predict_indices
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 975, in test_arma_predict_indices
    _check_start(*((model,) + case))
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 920, in _check_start
    assert_equal(start, expected)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arima_predict_indices
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 1047, in test_arima_predict_indices
    _check_start(*((model,) + case))
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 920, in _check_start
    assert_equal(start, expected)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

======================================================================
ERROR: statsmodels.tsa.tests.test_arima.test_arima_wrapper
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tsa/tests/test_arima.py", line 1572, in test_arima_wrapper
    'ma.L1.D.cpi']))
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 35, in assert_equal
    if not is_pandas_min_version('0.14.1'):
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_5/lib/statsmodels/tools/testing.py", line 20, in is_pandas_min_version
    from pandas.version import short_version as pversion
ImportError: No module named 'pandas.version'

```
statsmodel-0.6.1

**Comment by josef-pkt** [comment]
Thanks, 
I just saw this last night on our Travis testing, and I'm looking into it.

**Comment by jlec** [comment]
quick workaround could be

```patch
 setup.py                     | 2 +-
 statsmodels/tools/testing.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 1a7da9a..a6d1b33 100644
--- a/setup.py
+++ b/setup.py
@@ -134,7 +134,7 @@ def check_dependency_versions(min_versions):
                               (spversion, min_versions['scipy']))
 
     try:
-        from pandas.version import short_version as pversion
+        from pandas import __version__ as pversion
     except ImportError:
         install_requires.append('pandas')
     else:
diff --git a/statsmodels/tools/testing.py b/statsmodels/tools/testing.py
index 1fde1de..92e77fc 100644
--- a/statsmodels/tools/testing.py
+++ b/statsmodels/tools/testing.py
@@ -17,7 +17,7 @@ def strip_rc(version):
 def is_pandas_min_version(min_version):
     '''check whether pandas is at least min_version
     '''
-    from pandas.version import short_version as pversion
+    from pandas import __version__ as pversion
     return StrictVersion(strip_rc(pversion)) >= min_version
 
 
```

**Comment by PierreBdR** [comment]
This should be corrected by PR #2679

**Comment by josef-pkt** [comment]
closing, should be fixed, we haven't seen a problem anymore

thanks @PierreBdR
